operations/src/main/java/edu/unc/lib/boxc/operations/impl/download/DownloadBulkRequest.java

```diff
@@ -1,17 +1,17 @@
 package edu.unc.lib.boxc.operations.impl.download;
 
 import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import edu.unc.lib.boxc.auth.api.models.AccessGroupSet;
 import edu.unc.lib.boxc.auth.api.models.AgentPrincipals;
 import edu.unc.lib.boxc.auth.fcrepo.models.AgentPrincipalsImpl;
 
 public class DownloadBulkRequest {
     private String workPidString;
-    @JsonDeserialize(as = AgentPrincipalsImpl.class)
-    private AgentPrincipals agent;
+    private AccessGroupSet principals;
```

Comment:
Just curious why this change was needed? The AgentPrincipals object has the same info in it, plus the user name

Reply:
oh, i just changed it because the access check asked for the principals group set and i thought if i just required that then it would skip a method step, since i just kept converting back and forth